backend/attendance/CHANGELOG.md

```
@@ -1,6 +1,22 @@
+### Changes
+
+- Added new tag field in eg_wms_attendance_attendee table with supporting index 
+- Introduced support for attendee tagging and tag-based filtering
+- Enhanced attendee retrieval logic to support:
+    - Filtering attendees based on explicit tags
+    - Resolving and including attendees with matching tags when includeTaggedAttendees is true
+- Added new /_updateTag API to update attendee tags
+
+### Configuration Enhancements
+- Extended AttendanceRegisterSearchCriteria with:
+    - tags – list of tags to filter attendees
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)